R/caretEnsemble.R

```diff
@@ -142,7 +164,7 @@ predict.caretEnsemble <- function(object, keepNA = TRUE, se = FALSE, return_weig
       message("Predictions being made only for cases with complete data")
     }
     est <- as.numeric(preds %*% object$weights)
-    se.tmp <- apply(preds, 1, FUN = wtd.sd, weights = object$weights, normwt = TRUE)
+    se.tmp <- apply(preds, 1, FUN = wtd.sd, w = object$weights)
   } else if(keepNA == FALSE){
     if(anyNA(preds)){
```

Comment:
Finding a test case that meets this condition might not be possible -- I'd need to find a way to generate a model that predicts missing values for some observations, but is still ensembled.